src/ContactFormlet/ContactFormlet.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+
+function ContactFormlet({contacts, contactType, handleContactChange, contactData}) {
+  const formletOptions = {
+    owner: {
+      title: 'Owner'
+    },
+    vet: {
+      title: 'Vet'
+    },
+    farrier: {
+      title: 'Farrier'
+    }
+  }
+  const [isNew, setIsNew] = useState(false);
+
+  const options = contacts.map(contact => {
```

Comment:
I love this idea of selecting from the list of existing contacts or adding a new one.